include/swift/AST/Decl.h

```diff
@@ -1981,10 +1985,12 @@ class PatternBindingEntry {
     InitExpr.EqualLoc = equalLoc;
   }
 
-  /// Retrieve the initializer as it was written in the source.
-  Expr *getInitAsWritten() const {
-    return InitContextAndIsText.getInt() ? nullptr : InitExpr.Node;
-  }
+  /// Retrieve the initializer after the =, if any, as it was written in the
+  /// source.
+  Expr *getOrigInit() const;
```

Comment:
Nitpick: I actually like the old name better, but if you're not going to use it can you spell out "original"? (You could argue that "initializer" or "initial value" should be spelled out too, but at least that one's more common in the codebase already *and* has a domain-specific meaning. We shouldn't be shortening normal words when used for their typical meaning.)

Reply:
Thanks. There was an "Orig" before, and also "AsWritten", and I didn't like the potential confusion. I asked, but didn't get a good answer on the difference. (If you ask me, I would pick "AsParsed".) Since IIRC, there was an "Orig" before, I'll go with "Original". I agree with spelling out!